### DISCUSSION TITLE
Cuantos mensajes se pueden mandar antes de que Whatsapp bloquee el numero

### DISCUSSION THREAD

**Discussion by scullyiron** [codigoencasa/builderbot#122] [Q&A]
Mi duda radica en que la documentacion de la API de WhatsApp, dice que se va subiendo de nivel segun sea el numero de mensajes por cada 24hrs. Este Bot tiene alguna restricción de uso?

**Comment by NicoPauer** [comment]
Depende más de que tan seguido los envié pero por lo general alrededor de 5 el usuario se cansará de recibir tantos mensajes de un bot y bloquearía el número para dejar de recibirlos
